handler/handler.go

```diff
@@ -110,6 +110,7 @@ func (h *Handler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 		User:      "",
 		Program:   "",
 		Release:   "",
+		Selected:  r.URL.Query().Get("select"),
```

Comment:
As select is reserved keyword in golang, used selected and user can provide select as params.